lib/VerifiableCredentialStore.js

```diff
@@ -40,28 +40,27 @@ export default class VerifiableCredentialStore {
   /**
    * Gets all verifiable credential instances that match the given parameters
    *
-   * @param {string} [type]
-   * @param {string} [issuer]
-   * @param {boolean} [displayable]
+   * @param {object|Array} query
    *
    * @return {Promise<Array>} List of matching VCs
    */
-  async find({type, issuer, displayable}) {
-    const equals = {};
-    if(type) {
-      if(Array.isArray(type)) {
-        // needs to be implemented on the edv-client side first
-        throw new Error('"type" as array is not implemented.');
-      } else {
-        equals['content.type'] = type;
+  async find({query} = {}) {
```

Comment:
this is setup to allow `query` to be `undefined`, but this code breaks on L50 if query === `undefined`.  It looks to me that query is a required parameter.

Reply:
Yeah, we could do `{query = []} = {}` to set it by default, but better to throw if it isn't there. Good catch.